dask/values.yaml

```diff
@@ -131,6 +131,76 @@ worker:
       podTargetLabels: [] # PodTargetLabels transfers labels on the Kubernetes Pod onto the target.
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
 
+additional_worker_groups:
+  - worker:
+      name: worker  # Dask worker name.
+      image:
+        repository: "daskdev/dask"  # Container image repository.
+        tag: 2022.1.0  # Container image tag.
+        pullPolicy: IfNotPresent  # Container image pull policy.
+        dask_worker: "dask-worker"  # Dask worker command. E.g `dask-cuda-worker` for GPU worker.
+        pullSecrets:  # Container image [pull secrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+        #  - name: regcred
+      replicas: 3  # Number of workers.
+      strategy:
+        type: RollingUpdate  # Strategy used to replace old Pods with new ones.
+      custom_scheduler_url: null # connect to already existing scheduler, deployed not by this chart.
+      default_resources:  # overwritten by resource limits if they exist
+        cpu: 1  # Default CPU (DEPRECATED use `resources`).
+        memory: "4GiB"  # Default memory (DEPRECATED use `resources`).
```

Comment:
Given that these are deprecated options we should probably just not include them here.